### ISSUE TITLE
Add "breathing space" in the design.

### ISSUE THREAD

**Issue by programnoir** [nitronova/StudyGameDev.com#22]
One critique from the author of the accessible menu that the one in StudyGameDev was based on, Marcy Sutton, was that the design overall lacked "breathing space." It is described in better detail here: https://alvalyn.com/breathing-room-the-role-of-the-space-in-between/

White space in the design needs to serve two main roles:
1. In making the interactive parts of the site easier to hit (and harder to miss) in visual and tactile terms.
2. Adding emphasis (and reducing noise) surrounding elements of interest.
